### PR TITLE
Gemfile changes for 6.3.1

### PR DESCRIPTION
**Pull request by robbavey**

### CODE REVIEW

File: Gemfile.jruby-2.3.lock.release [hunk 1]
```diff
@@ -287,9 +286,8 @@ GEM
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       rufus-scheduler
       stud (~> 0.0.22)
-    logstash-input-file (4.0.5)
+    logstash-input-file (4.1.2)
```
Comment (jakelandis) [inline Gemfile.jruby-2.3.lock.release]: @guyboertje - do you feel comfortable enough with the passivity of this to upgrade this in a patch release ?

File: Gemfile.jruby-2.3.lock.release [hunk 2]
```diff
@@ -312,12 +310,10 @@ GEM
       logstash-codec-plain
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       stud
-    logstash-input-http (3.0.10)
+    logstash-input-http (3.2.0-java)
```
Comment (jakelandis) [inline Gemfile.jruby-2.3.lock.release]: @jsvd - do you feel comfortable enough with the passivity of this to upgrade in a patch release ?

File: Gemfile.jruby-2.3.lock.release [hunk 3]
```diff
@@ -114,13 +113,13 @@ GEM
...
     jar-dependencies (0.3.12)
-    jls-grok (0.11.4)
+    jls-grok (0.11.5)
       cabin (>= 0.6.0)
     jls-lumberjack (0.0.26)
       concurrent-ruby
     jmespath (1.4.0)
     jrjackson (0.4.6-java)
-    jruby-openssl (0.9.21-java)
+    jruby-openssl (0.10.0-java)
```
Comment (jakelandis) [inline Gemfile.jruby-2.3.lock.release]: 🤞

File: Gemfile.template [hunk 4]
```diff
@@ -38,7 +38,8 @@ gem "logstash-codec-json_lines"
 gem "logstash-codec-line"
 gem "logstash-codec-msgpack"
 gem "logstash-codec-multiline"
-gem "logstash-codec-netflow"
+# Pin to 3.x branch to avoid breaking change in Logstash 6.x
```
Comment (jakelandis) [inline Gemfile.template]: We have changes in this codec only compatible with Logstash 7.x ?

**Comment by jakelandis** [comment]
Looking good... a couple outstanding questions before merging.

File: Gemfile.template (hunk 4 above)
Comment (robbavey) [reply Gemfile.template]: No not Logstash 7.x, but not BWC - I'll change the comment to be less ambiguous https://github.com/logstash-plugins/logstash-codec-netflow/blob/master/CHANGELOG.md

File: Gemfile.jruby-2.3.lock.release (hunk 1 above)
Comment (jsvd) [reply Gemfile.jruby-2.3.lock.release]: let's skip this

File: Gemfile.jruby-2.3.lock.release (hunk 2 above)
Comment (jsvd) [reply Gemfile.jruby-2.3.lock.release]: let's skip this

**Pull request closed without merging**